.gitignore

```diff
@@ -6,3 +6,4 @@ default.etcd
 oauth/client.json
 oauth/user.json
 .idea
+example.eskip
```

Comment:
The readme already suggests how to create this particular file. If we decide to commit one such file we'll change the readme